package.json

```diff
@@ -45,7 +45,6 @@
     "npm-run-all": "^4.1.1",
     "nsp": "^2.8.1",
     "onchange": "^3.2.1",
-    "path": "^0.12.7",
```

Comment:
We use this devDependency in [run-firefox.js](https://github.com/mozilla/shield-studies-addon-template/blob/master/run-firefox.js#L13); can you add it back?

Reply:
That removal comes from https://github.com/mozilla/shield-studies-addon-template/pull/41 - comment that this dependency is used, close the pr and I will rebase this pr to not include this removal :)

Reply:
The [`path` module is the built in Node.js module](https://nodejs.org/dist/latest-v8.x/docs/api/path.html). Adding it in package.json includes a module from [npm](https://www.npmjs.com/package/path) &mdash; which could potentially be dangerous.

Reply:
Noted, and I have merged #41. Thank you Peter for clarifying!